cli/internal/runcache/runcache.go

```diff
@@ -41,6 +42,73 @@ type Opts struct {
 	LogReplayer       LogReplayer
 }
 
+// AddFlags adds the flags relevant to the runcache package to the given FlagSet
+func AddFlags(opts *Opts, flags *pflag.FlagSet) {
+	flags.BoolVar(&opts.SkipReads, "force", false, "Ignore the existing cache (to force execution).")
+	flags.BoolVar(&opts.SkipWrites, "no-cache", false, "Avoid saving task results to the cache. Useful for development/watch tasks.")
+	flags.AddFlag(&pflag.Flag{
+		Name:     "output-logs",
+		Usage:    _outputModeHelp,
+		DefValue: "full",
+		Value:    &logsModeValue{opts: opts},
+	})
+	_ = flags.Bool("stream", true, "Unused")
+	if err := flags.MarkDeprecated("stream", "[WARNING] The --stream flag is unnecesary and has been deprecated. It will be removed in future versions of turbo."); err != nil {
+		// fail fast if we've misconfigured our flags
+		panic(err)
```

Comment:
Yeah. Unfortunately due to how the flags api works, there are some methods that return errors when they can't find the specified flag, as opposed to being able to set attributes on the flag declaration itself. I put those as panics since they should all be executed synchronously from `main`, so it should fail quickly and consistently if it's broken.